packages/math-input/src/components/key-translator.ts

```diff
@@ -99,21 +125,19 @@ const keyToMathquillMap: Record<Key, MathQuillCallback | null> = {
 
     // These need to be overwritten by the consumer
     // if they're going to be used
-    CUBE_ROOT: null,
-    FRAC: null,
-    FRAC_EXCLUSIVE: null,
-    RIGHT: null,
-    LEFT: null,
-    BACKSPACE: null,
-    DISMISS: null,
-    JUMP_OUT_PARENTHESES: null,
-    JUMP_OUT_EXPONENT: null,
-    JUMP_OUT_BASE: null,
-    JUMP_INTO_NUMERATOR: null,
-    JUMP_OUT_NUMERATOR: null,
-    JUMP_OUT_DENOMINATOR: null,
-    NOOP: null,
-    MANY: null,
+    FRAC: () => {},
```

Comment:
I realized that TS doesn't require you to have the params if you're not using them, so it seemed to make sense to just use a noop here. Also I was running into something that made me think TS was casting `MathQuillUpdateCallback | null` to `any`, but I could have just been spacing.